frontend/src/components/ui/button.ts

```diff
@@ -35,7 +35,7 @@ export class Button extends TailwindElement {
   filled = false;
 
   @property({ type: String })
-  size: "small" | "medium" = "medium";
+  size: "xs" | "sm" | "md" = "md";
```

Comment:
Nitpick but would be nice to standardize around shoelace size so we don't need to memorize two separate size attributes

Reply:
Ahh, fair — this was based on Tailwind sizes. Happy to update to Shoelace though!

Reply:
Yeah, might be the way to go, since we do use shoelace size a lot for inputs, buttons, etc.

Reply:
Oh, hmm, does shoelace have an `xs` equivalent?